components/ide/jetbrains/launcher/main.go

```diff
@@ -637,6 +642,25 @@ func handleSignal() {
 	log.Info("asked IDE to terminate")
 }
 
+func updatePlatformProperties(platformOptionsPath string, configDir string, systemDir string) error {
```

Comment:
this function should make sure that it does not change file unnecessary if we restart JB backend, i.e. kill lunacher process